tanjun/components.py

```diff
@@ -923,7 +923,8 @@ async def _execute_interaction(
 
         except errors.CommandError as exc:
             await ctx.respond(exc.message)
-            return asyncio.get_running_loop().create_future().set_result(None)
+            asyncio.get_running_loop().create_future().set_result(None)
+            return None
```

Comment:
mypy has a stupid thing where it says functions that return `None` don't actually return any value at all. I haven't noticed this before, mostly because it's not a thing in async functions: https://github.com/python/mypy/issues/9191

Talk about consistent!

Reply:
don't need the None in that return statement even if mypy complains

Reply:
mypy doesn't actually let you drop `None` from `return`, quite opinionated. "explicit > implicit" or smth.

https://github.com/python/mypy/issues/7511

Reply:
Silence it then 🔫👏

Reply:
I don't think this is too much of a problem tbf, and AFAIK there's no way to silence just this (eventuallytm there'll be like, proper error codes in mypy and that'll be nice)